doc/neobundle.txt

```diff
@@ -383,10 +383,11 @@ neobundle#has_fresh_cache()	 *neobundle#has_fresh_cache()*
 		Note: It is deprecated.  You should use
 		|neobundle#load_cache()| instead of it.
 
-neobundle#load_cache([{vimrc}])			 *neobundle#load_cache()*
+neobundle#load_cache([{vimrcs}])		 *neobundle#load_cache()*
 		Load plugin configuration from the cache file,
 		which is located in `neobundle#get_rtp_dir() . '/cache'`.
-		{vimrc} is compared .vimrc file.  The default is |$MYVIMRC|.
+		{vimrcs} is a list of compared .vimrc and/or other configuration
```

Comment:
I'm not sure whether it's right to use the term "list" here, because this function does not take a `list` like `call neobundle#load_cache([$MYVIMRC, 'plugin.vim'])` actually.